docs/UserGuide.adoc

```diff
@@ -670,6 +688,14 @@ the command will not be able to undo or redo.
 ====
 // end::undoredo[]
 
+[NOTE]
+====
+Redoable commands: those commands that modify CHRS content.
+For commands similar functions to +
+(list*, find*, filter*, select*, calculate*)
+the command will not be able to undo or redo.
+====
+
```

Comment:
This note is also a repeated note within the userguide, there already exists one (exact same) for redo